docs-new/docs/graphql/core/scheduled-triggers/create-one-off-scheduled-event.mdx

```diff
@@ -0,0 +1,158 @@
+---
+description: Create a one-off scheduled event with Hasura
+keywords:
+  - hasura
+  - docs
+  - one off scheduled event
+  - scheduled trigger
+  - create
+sidebar_position: 2
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import Thumbnail from "@site/src/components/Thumbnail/Thumbnail";
+
+# Creating a one-off scheduled event
+
+## Introduction
+
+One-off scheduled events are used to reliably trigger an HTTP webhook to run custom business logic at a particular point in time. For example, you can create a scheduled event to send a reminder email two weeks after a user signs up.
+
+To add a one-off scheduled event, follow these steps:
+
+## Step 1: Navigate to One-off Scheduled Events
+
+- Go to the `Events` tab in your Hasura console.
+- Click `One-off Scheduled Events`.
+- Click `Schedule an event`.
+
+<Thumbnail
+  src="/img/graphql/core/event-triggers/one-off.png"
+  alt="Adding a one-off scheduled event"
+  width="900px"
+/>
+
+## Step 2: Define the scheduled event
+
+Define the following values for a scheduled event:
+
+- **Webhook**: Enter the HTTP endpoint that should be triggered.
+- **Time**: Enter the time to trigger the event.
+- **Payload**: The JSON payload which will be sent to the webhook.
+
+<Tabs className="api-tabs">
+<TabItem value="console" label="Console">
+In the form opened by the above step, fill out the following fields:
+
+<Thumbnail
+  src="/img/graphql/core/event-triggers/define-one-off-event.png"
+  alt="Defining the scheduled event"
+  width="550px"
+/>
+</TabItem>
+<TabItem value="cli" label="CLI">
+
+One-off scheduled events cannot be set using the CLI as they are not tracked as a part of Hasura metadata.
+
+</TabItem>
+<TabItem value="api" label="API">
+
+You can define a scheduled event via the [create_scheduled_event](/graphql/core/api-reference/metadata-api/scheduled-triggers.mdx#metadata-create-scheduled-event) metadata API:
+
+```http
+POST /v1/metadata HTTP/1.1
+Content-Type: application/json
+X-Hasura-Role: admin
+
+{
+   "type": "create_scheduled_event",
+   "args": {
+      "webhook": "https://send-email.com",
+      "schedule_at": "2022-06-18T18:45:00Z",
+      "payload": { "email": "bob@ross.com" }
+   }
+}
+```
+
+</TabItem>
+</Tabs>
+
+## Step 3: Define advanced options (Optional)
+
+If you like, you can also define advanced values:
+
+-   **Headers**: List of headers to be sent to the webhook.
+-   **Retry configuration**: In case the call to the webhook fails.
+-   **Comment**: Custom description of the scheduled trigger.
+
+<Tabs className="api-tabs">
+<TabItem value="console" label="Console">
+
+Expand the `Advanced` section.
+
+<Thumbnail
+  src="/img/graphql/core/event-triggers/advanced-one-off.png"
+  alt="Defining the scheduled event"
```

Comment:
Mismatched alt text - please refer original file and correct this. Also please check for other occurrences of any mismatch in this file

Reply:
Ack.